leaper.py

```diff
@@ -135,7 +135,7 @@ def rdiff_link(self, src_project, src_package, src_rev, target_project, target_p
                 }
 
     def check_source_submission(self, src_project, src_package, src_rev, target_project, target_package):
-        self.logger.info("%s/%s@%s -> %s/%s"%(src_project, src_package, src_rev, target_project, target_package))
+        super(Leaper, self).check_source_submission(src_project, src_package, src_rev, target_project, target_package)
```

Comment:
Yeah, I looked it up a while back since it seemed rather ugly, but came to the same conclusion although type(self) is arguably more flexible.